test/health-check.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -Eeuo pipefail
+
+export AWS_DEFAULT_REGION=${AWS_DEFAULT_REGION:-ap-southeast-2}
+export AWS_DEFAULT_OUTPUT=${AWS_DEFAULT_OUTPUT:-text}
+
+IPS=$(aws ec2 describe-instances --filters "Name=tag:envid,Values=${ENVID:?}" --query "Reservations[*].Instances[*].PublicIpAddress" | tr '\t' '\n')
+test -n "$IPS" || echo "No instances found with envid: ${ENVID} in ${AWS_DEFAULT_REGION} region"
```

Comment:
So if IPS is empty this script succeeds?

Reply:
I would expect this checking number of nodes.

Reply:
Good point, @valerifilipov do you mind improving it?